core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -2252,18 +2252,47 @@ private long loadMetadataAndJournal(LockedInodePath inodePath, LoadMetadataOptio
       throws InvalidPathException, FileDoesNotExistException, BlockInfoException,
       FileAlreadyCompletedException, InvalidFileSizeException,
       AccessControlException, IOException {
+    return loadMetadataAndJournalInternal(inodePath, options, LoadMetadataStatus.UNKNOWN);
+  }
+
+  /**
+   * Internal implementation for
+   * {@link #loadMetadataAndJournal(LockedInodePath, LoadMetadataOptions)}.
+   *
+   * @param inodePath the path for which metadata should be loaded
+   * @param options the load metadata options
+   * @param loadMetadataStatus status of path in UFS if known
```

Comment:
if this is specific to the ufs path, should this be called something like `ufsMetadataStatus`?